README.md

```diff
@@ -37,9 +37,32 @@ async function createIssueOnAllRepos (org) {
 
 Handle events
 
+Return `true` if you wish to retry the request, it will be retried after `retryAfter` seconds.
+
 ```js
-octokit.throttle.on('rate-limit', (retryAfter) => console.warn(`Rate-limit hit, retrying after ${retryAfter}s`))
-octokit.throttle.on('abuse-limit', (retryAfter) => console.warn(`Abuse-limit hit, retrying after ${retryAfter}s`))
+const Octokit = require('@octokit/rest')
+  .plugin(require('@octokit/plugin-throttling'))
+
+const octokit = new Octokit({
+  throttle: {
+    onRateLimit: (retryAfter, options) => {
+      console.warn(`Rate-limit hit for request ${options.method} ${options.url}`)
+
+      // Ex.: only retry twice
+      if (options.request.retryCount < 2) {
+        return true
+      }
+    },
+    onAbuseLimit: (retryAfter, options) => {
+      console.warn(`Abuse-limit hit for request ${options.method} ${options.url}`)
+
+      // Ex.: only retry GET requests
+      if (options.method === 'GET') {
+        return true
+      }
```

Comment:
I don’t think you can hit abuse limits with `GET` requests. I don’t have an idea for an alternative example right now, maybe you have one?